.vscode/launch.json

```diff
@@ -166,11 +166,15 @@
             "type": "node",
             "request": "launch",
             "autoAttachChildProcesses": true,
-            "skipFiles": ["<node_internals>/**", "**/node_modules/**"],
+            "skipFiles": ["<node_internals>/**", "**/node_modules/**", "!**/node_modules/vscode-uri/**"],
             "program": "${workspaceRoot}/node_modules/vitest/vitest.mjs",
             "args": ["run", "${relativeFile}"],
             "smartStep": true,
             "console": "integratedTerminal",
+            "sourceMaps": true,
+            "outFiles": [
+                "${workspaceFolder}/node_modules/vscode-uri/lib/**/*.js",
+            ]
```

Comment:
The `outFiles` property on here behaves as a filter not as an addition to the existing outfiles. Meaning that the only `outFiles` being considered are now those from `node_modules/vscode-uri`. This prevents debugging the Langium source code.

Reply:
I cannot confirm that, see below. However, the source maps generated by Vitest/Vite on the fly are out-of-sync 😖, see also https://github.com/eclipse-langium/langium/pull/1814#discussion_r1963601132
![source-maps-out-of-sync](https://github.com/user-attachments/assets/4577c992-74e8-439b-a296-ca98220c5d55)

Reply:
Interesting. Yes, I can confirm that keeping it empty also works for me 👍